src/GitHub.App/Services/ModelService.cs

```diff
@@ -125,8 +126,12 @@ IObservable<IEnumerable<AccountCacheItem>> GetUserOrganizations()
         public IObservable<IReadOnlyList<IRepositoryModel>> GetRepositories()
         {
             return GetUserRepositories(RepositoryType.Owner)
-                .Take(1)
-                .Concat(GetUserRepositories(RepositoryType.Member).Take(1))
+                .PublishLast()
```

Comment:
These methods only returns either 1 or 2 results (depending on whether they only return from cache or they hit the cache and then the server).

`Take(1)` implies that we only want the first result, so the code that refreshes the cache from the server never runs if there's anything in the cache to return, because it's all run on demand (and we never demand all the results back)

`PublishLast()` is an async `Last()` method, which is what we actually want (to return the most up to date result).